python/lsst/summit/utils/bestEffort.py

```diff
@@ -27,7 +27,7 @@
 from lsst.daf.butler.registry import ConflictingDefinitionError
 
 from lsst.summit.utils.quickLook import QuickLookIsrTask
-from lsst.summit.utils.butlerUtils import getLatissDefaultCollections
```

Comment:
There's the linter error (this is as close as I can put a comment I think), but otherwise this looks fine.